ostree/README.md

```diff
@@ -14,33 +14,41 @@ Scripts are provided for both of these, and instructions for running directly or
 The createOSTreeUpdate.sh script can be used to create a field upgrade tarball.
 
 ```
-> ./createOSTreeUpdate <old-wic> <new-wic> [upgrade-tag]
+> ./createOSTreeUpdate.sh old-wic-file new-wic-file [upgrade-tag]
+```
+
+If using the Docker container use:
+
+```
+docker build --no-cache -f Docker/Dockerfile --label ostree-delta  --tag ${USER}/ostree-delta:latest .
+docker run --rm -v old_wic_file:/old_wic.wic -v new_wic_file:new_wic.wic -v /dev:/dev -v ${PWD}:/ws -w /ws --privileged ${USER}/ostree-delta:latest ./createOSTreeUpgrade.sh /old_wic.wic /new_wic.wic [upgrade-tag]
```

Comment:
Why is '--privileged' needed?

Reply:
Without it the mount of the loop device fails.
```
Mounting wic partition 2 of /tmp/tmp.jLdgG6Gizu/old_wic to /tmp/tmp.jLdgG6Gizu/old
mount: /tmp/tmp.jLdgG6Gizu/old: failed to setup loop device for /tmp/tmp.jLdgG6Gizu/old_wic.
```

Reply:
Might be good to comment that there?

Reply:
This PR has already been merged. 
Please re-check in https://github.com/PelionIoT/scripts-pelion-edge/pull/20